packages/template-chakra-storefront/src/components/list-menu/list-menu.test.js

```diff
@@ -16,22 +16,25 @@ describe('ListMenu', () => {
         const user = userEvent.setup()
         renderWithProviders(<ListMenu root={mockCategories.root} itemsKey="categories" />)
 
```

Comment:
Would you mind adding tests for keyboard a11y cases?

Reply:
@alexvuong 🤔 what are some examples of these a11y cases?